frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -1614,6 +1618,11 @@ private void prepareDetailView(RenderRequest request, RenderResponse response) t
                 List<ProjectLink> mappedProjectLinks = createLinkedProjects(project, user);
                 request.setAttribute(PROJECT_LIST, mappedProjectLinks);
                 List<ProjectLink> allSubProjectLinks = createLinkedProjects(project, Function.identity(), true, user);
+                Set<ProjectLink> subprojects = allSubProjectLinks.parallelStream().map(l -> l.getSubprojects()).flatMap(l-> l.stream())
```

Comment:
convert `List<ProjectLink> allSubProjectLinks` to `Map<String, ProjectLink>` and pass the `values()` of Map in request attribute. <br>

Kindly revert unwanted changes in Line # `102` and `1635` 
fix indentation on line # `2127`